module/zstd/Makefile.in

```diff
@@ -0,0 +1,36 @@
+src = @abs_top_srcdir@/module/zstd
+obj = @abs_builddir@
+
+MODULE := zzstd
+
+obj-$(CONFIG_ZFS) := $(MODULE).o
+
+asflags-y += $(ZFS_MODULE_CFLAGS)
+ccflags-y := $(ZFS_MODULE_CFLAGS) $(ZFS_MODULE_CPPFLAGS)
+
+# Suppress unused but set variable warnings often due to ASSERTs
+ccflags-y += $(NO_UNUSED_BUT_SET_VARIABLE)
+ccflags-y += -Wframe-larger-than=2048 -O3
```

Comment:
2048 or 20480? are you sure -O3 is a good idea?

Reply:
Jude has:
```
ccflags-y += -I@abs_top_srcdir@/module/zstd/include
ccflags-y += -Wframe-larger-than=20480 -Wno-uninitialized
```

Reply:
sure its a good idea. factory optimization of zstd and frame larger etc is a requirement and must be minimum somewhere at 1024 - 1100 depending on the gcc version, but this counts only for x86. on other architectures it must be higher